doc/api/deprecations.md

```diff
@@ -3019,6 +3019,17 @@ should have the same effect. The receiving end should also check the
 [`readable.readableEnded`][] value on [`http.IncomingMessage`][] to get whether
 it was an aborted or graceful destroy.
 
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/40773
+    description: Documentation-only deprecation.
+-->
+
+Type: Documentation-only
```

Comment:
This says `Documentation-only` but there are significant code changes....is this an oversight?